<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sk2_assembler</class>
 <widget class="QMainWindow" name="sk2_assembler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="grupo_superior">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="converter">
      <property name="text">
       <string>Converter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="tudo">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="assembly_grupe">
         <property name="title">
          <string>Assembly</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPlainTextEdit" name="caixa_texto_assembly">
            <property name="placeholderText">
             <string>Digite o código assembly aqui...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="carregar">
            <property name="text">
             <string>Carregar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="binary_grupe">
         <property name="title">
          <string>Binário</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPlainTextEdit" name="caixa_texto_binario">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>O código binário aparecerá aqui...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="contador_bytes">
            <property name="text">
             <string>Bytes: 0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="botoes_binary_grupe">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="copiar">
               <property name="text">
                <string>Copiar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="salvar">
               <property name="text">
                <string>Salvar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
